jmxtrans-docker-test/pom.xml

```diff
@@ -0,0 +1,426 @@
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jmxtrans</groupId>
+        <artifactId>jmxtrans-parent</artifactId>
+        <version>254-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jmxtrans-docker-test</artifactId>
+
+    <properties>
+        <docker.verbose>true</docker.verbose>
+        <verification.message>installation verified</verification.message>
+        <verification.wait.time>30000</verification.wait.time>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jolokia</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.13.8</version>
+                <configuration>
+                    <showLogs>true</showLogs>
+                    <images/>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>start</id>
+                        <goals>
+                            <goal>build</goal>
+                            <goal>start</goal>
+                        </goals>
+                        <phase>pre-integration-test</phase>
+                    </execution>
+                    <execution>
+                        <id>stop</id>
+                        <goals>
+                            <goal>stop</goal>
+                        </goals>
+                        <phase>post-integration-test</phase>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>deb</id>
+
+            <properties>
+                <deb.install.artifact>org.jmxtrans:jmxtrans:deb</deb.install.artifact>
+                <deb.install.file>jmxtrans.deb</deb.install.file>
+                <deb.run.cmd>dpkg -i /maven/${deb.install.file}; /etc/init.d/jmxtrans start; /maven/verify.sh</deb.run.cmd>
+            </properties>
+
+            <dependencies>
+                <dependency>
+                    <groupId>${project.groupId}</groupId>
+                    <artifactId>jmxtrans</artifactId>
+                    <version>${project.version}</version>
+                    <type>deb</type>
+                </dependency>
+            </dependencies>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.jolokia</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+
+                        <configuration>
+                            <images combine.children="append">
+                                <image>
```

Comment:
Yes, I'm not quite happy with amount of duplication but I've tried to minimize it as much as possible.  I'm brainstorming on ways to improve this but I wanted to start testing this on Travis first while I had workable solution rather than continue to gold plate it.